.github/workflows/cron-weekly.yml

```diff
@@ -24,3 +24,18 @@ jobs:
           use-verbose-mode: 'yes'
```

Comment:
tying all these workflows to the same exact schedule seems brittle to me. Can we keep these as separate files?

Reply:
isn't this the only workflow on a schedule? consolidating to our "weekly" checks seems like a positive IMO

Reply:
not anymore https://github.com/GoogleChrome/lighthouse/blob/master/.github/workflows/publish.yml#L4